environments/openstack/playbook-bootstrap-basic.yml

```diff
@@ -291,6 +291,31 @@
     - name: Make ubuntu minimal 20.04 image visible
       ansible.builtin.command: openstack --os-cloud admin image set --public 'Ubuntu 20.04'  # noqa 301
 
+    - name: Download amphora image
+      ansible.builtin.get_url:
+        url: https://minio.services.osism.tech/openstack-octavia-amphora-image/octavia-amphora-haproxy-xena.qcow2
+        dest: /tmp/amphora.img
+
+    - name: Upload amphora image
+      openstack.cloud.image:
+        cloud: octavia
+        state: present
+        name: amphora
+        container_format: bare
+        disk_format: qcow2
+        filename: /tmp/amphora.img
+        min_disk: 2
+        tags:
+          - amphora
+        properties:
+          cpu_arch: x86_64
+          distro: ubuntu
+          hw_rng_model: virtio
+
+    # NOTE: is_public from os_image is not working like expected
+    - name: Make amphora image visible
+      ansible.builtin.command: openstack --os-cloud octavia image set --public 'amphora'  # noqa 301
```

Comment:
Why should the image be public? IIUC it should only be used by the octavia service and not be visible to other tenants.

Reply:
Private is sufficient, yes.